File: ucare_backend/src/main/java/com/douzone/ucare/service/ReceiptService.java
package com.douzone.ucare.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.douzone.ucare.repository.ReceiptRepository;
import com.douzone.ucare.vo.ReceiptVo;

@Service
public class ReceiptService {
	
	@Autowired
	private ReceiptRepository ReceiptRepository;

	public int create(ReceiptVo receipt) {
		return ReceiptRepository.create(receipt);
	}

	public List<ReceiptVo> retrieveAll(Long patientNo) {
		return ReceiptRepository.retrieveAll(patientNo);
	}
	
	public int delete(Long receiptNo) {
		return ReceiptRepository.delete(receiptNo);
	}
}

File: ucare_frontend/src/service/receiptService.js
import axios from 'axios';
import { RECEIPT_API_BASE_URL } from './urlConfig';

class receiptService {
  
  create(data){
    return axios.post(RECEIPT_API_BASE_URL + '/create', data);
  }

  retrieveAll(patientNo){
    return axios.get(RECEIPT_API_BASE_URL + '/retrieveAll/' + patientNo);
  }

  delete(receiptNo){
    return axios.delete(RECEIPT_API_BASE_URL + '/delete/' + receiptNo);
  }
}

export default new receiptService();